recipes/awsbatch_config.rb

```diff
@@ -42,9 +42,10 @@
     cwd Chef::Config[:file_cache_path]
     code <<-CLI
       set -e
-      curl --retry 3 -v -L -o aws-parallelcluster.tgz #{node['cfncluster']['custom_awsbatchcli_package']}
-      tar -xzf aws-parallelcluster.tgz
-      cd *aws-parallelcluster-*
+      curl --retry 3 -L -o aws-parallelcluster-custom-cli.tgz #{node['cfncluster']['custom_awsbatchcli_package']}
+      mkdir aws-parallelcluster-custom-cli
```

Comment:
[MINOR] can we have the same change (create dir) on the custom node installation?